.eslintrc.json

```diff
@@ -1,15 +1,22 @@
 {
-  "extends": ["eslint:recommended", "plugin:import/errors", "prettier"],
-  "plugins": ["import"],
```

Comment:
This plugin was added to lint the import structure (https://www.npmjs.com/package/eslint-plugin-import).  Why was it removed?

Reply:
Sorry. I think during the initial configuration I had a problem with it and forgot to add it back. I have reverted this change and everything looks to be fine. I have noticed that `import/errors` doesn't affect the order of imports though, for that `imports/order` would have to be activated.